README.md

```diff
@@ -201,334 +201,17 @@ Ideally, by the time of project submission, all smoke tests will pass.
 
 ## Functional Requirements
 
-Every endpoint must serve JSON data, and must use HTTP response codes to indicate the status of the request.
+All of the API requirements for this project are located in the project repo's wiki page.
```

Comment:
Just a suggested link

```suggestion
All of the API requirements for this project are located in the project repo's [wiki](https://github.com/AdaGold/video-store-api/wiki) page.
```

Reply:
@CheezItMan I love this! I'm under the assumption that the wikis get copied when you fork a repo, so I was wondering if it'd be more stable to always point them to their cohort wiki?

Reply:
Or does it not matter, and it's more important to point them to AdaGold's wiki?

Reply:
Ehhhh *checks clock* it's probably better to just go with this for now, but that's just an outstanding question I had!